llm_on_ray/finetune/data_preprocess.py

```diff
@@ -0,0 +1,229 @@
+#
+# Copyright 2023 The LLM-on-Ray Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import copy
+import re
+from itertools import chain
+
+import torch
+
+IGNORE_INDEX = -100
+
+
+class DataPreprocess:
```

Comment:
suggest to split this class two parts:
1. make prompt -> data_temmplate.py
2. tokenize -> data_process.py

Reply:
I have rename python file name and function name. But I still put the make_prompt and tokenzie function in a class. Beacuse both need to use common variables.

![image](https://github.com/intel/llm-on-ray/assets/45281494/a3e2829d-e695-4a17-b70b-a0d82bbb863a)